docs/source/onboarding_resources/computing/index.rst

```diff
@@ -280,22 +280,166 @@ lines from your trainings!
 Your Bash Configuration Files 
 -----------------------------
 
-Bash files contain commands you would enter into a command line, but 
-specifically ones you will use very often. For example, every time you 
-want to open a Jupyter session, the cluster needs certain information 
-and requirements. To find this information, it looks in your Bash files 
-rather than asking you to enter the same information every time. 
+Bash configuration files contain information and commands that are used when 
+interacting with the cluster. When the cluster tries to execute some command 
+line prompts, it will look to your bash configuration files for information 
+or filepaths. 
 
-However, Bash files can be confusing since it is less obvious when the 
-information is being used or what it is used for. Therefore, we have 
-provided a copy-and-paste formatting for information to be added to 
-your Bash files. 
+You should have 2 bash configuration files, your bash profile or :code:`.bash_profile` 
+and your bash rc or :code:`.bashrc`. 
 
-.. todo::
+Once you make edits to either of these files, you will need to log out and then back into 
+the cluster before they will take effect. 
 
-  - Confirm and paste in bash files 
-  - Get Zeb's help in explaining Bash files better 
-  - Moving up since this is needed for a lot of things below 
+bash profile
+~~~~~~~~~~~~
+
+The bash profile is only run when you open the cluster, and so is usually a shorter file. 
+Below are the lines of code you need in your bash profile. 
```

Comment:
Are these lines of code that you need to manually add to your `.bash_profile`, or should they already be there? I don't remember having to edit anything in my `.bash_profile` when first logging into the cluster. If it's things that are already there, perhaps we should clarify that you don't need to mess with it. Or even just don't go into any details if it's not something the user needs to know -- e.g., say something like "Your bash profile contains a few lines of code to load generic profile settings and load your `.bashrc` file. You shouldn't need to edit these." Or, if the precise lines of code _are_ something the user needs to know, clarify exactly what the user needs to do.

Reply:
Hmm good point! I edited this a bit to make it more clear that you shouldn't need to do anything for this one.